ddapm_test_agent/agent.py

```diff
@@ -243,6 +243,8 @@ def __init__(self) -> None:
             "/v0.1/pipeline_stats",
             "/tracer_flare/v1",
             "/evp_proxy/v2/api/v2/llmobs",
+            "/evp_proxy/v2/api/intake/llm-obs/v1/eval-metric",
```

Comment:
Do we care about adding this one?

Reply:
node.js sdk is still using it

Reply:
i do believe we need to switch it for node.js but it's sorta slipped my mind and hasn't been causing us any issues - if it's OK with you, I say we can keep this here for now!